src/main/flight/pid.h

```diff
@@ -98,7 +98,7 @@ typedef struct pidProfile_s {
     uint16_t itermThrottleThreshold;        // max allowed throttle delta before iterm accelerated in ms
     uint16_t itermAcceleratorGain;          // Iterm Accelerator Gain when itermThrottlethreshold is hit
     uint8_t setpointRelaxRatio;             // Setpoint weight relaxation effect
-    uint8_t dtermSetpointWeight;            // Setpoint weight for Dterm (0= measurement, 1= full error, 1 > aggressive derivative)
+    uint16_t dtermSetpointWeight;            // Setpoint weight for Dterm (0= measurement, 1= full error, 1 > aggressive derivative)
```

Comment:
Yes it does.

Reply:
Sorry I don't understand. Should I change something there?

Reply:
You need to change the version of this parameter group, because the format is changed (the version is the '2' at the end of https://github.com/betaflight/betaflight/blob/346201d8ac4704456bc9054104ad7bd3731cb2d0/src/main/flight/pid.c#L91).

Reply:
Thanks :)